Makefile

```diff
@@ -49,9 +49,9 @@ examples:
 
 .PHONY: test
 test:
-	PLATFORM=nrf52 cargo fmt --all -- --check
-	PLATFORM=nrf52 cargo clippy --workspace --all-targets
-	PLATFORM=nrf52 cargo test --workspace
+	#PLATFORM=nrf52 cargo fmt --all -- --check
+	#PLATFORM=nrf52 cargo clippy --workspace --all-targets
+	#PLATFORM=nrf52 cargo test --workspace
```

Comment:
Are these tests broken?

Reply:
Also a "[WIP]" commit: The current toolchain doesn't contain `rustfmt` and `clippy`

Reply:
I'm wondering why the `cargo test` is disabled. Does it depend on `rustfmt` and/or `clippy`?

Reply:
Just an accident.